CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# How to contribute
+
+All patches or feature evolutions are welcome.
+
+## Getting Started
+
+* Make sure you have a [GitHub account](https://github.com/signup/free)
+* Submit a ticket for your issue, assuming one does not already exist.
+  * Clearly describe the issue including steps to reproduce when it is a bug.
+  * Make sure you fill in the earliest version that you know has the issue.
+* Fork the repository on GitHub
+
+## Making Changes
+
+* Create a topic branch from where you want to base your work.
```

Comment:
Well, non-contributers won't be able to create topic branch. They will have to fork.